torch/csrc/jit/codegen/cuda/compute_at.cpp

```diff
@@ -472,7 +472,10 @@ unsigned int ComputeAt::backwardComputeAt_impl(
   }
 
   auto replay_producer_pair = TransformReplay::replayPasC(
-      producer, consumer, (int)consumer_compute_at_pos, root_map_);
+      producer,
+      consumer,
+      (int)consumer_compute_at_pos,
+      PairwiseRootDomainMap(producer, consumer));
```

Comment:
We don't need to use `ComputeAtRootDomaiMap` here, and it can fail to generate one-to-one mappings with views. `PairwiseRootDomainMap` is sufficient.